### DISCUSSION TITLE
watch mode Not effective

### DISCUSSION THREAD

**Discussion by znoliver** [eclipse-theia/theia#13419] [Q&A]
I created a browser project, and when I use watch for development，tsc can incremental compilation,but the browser project don't have response, did I make a mistake somewhere?

### My script
```
"scripts": {
    "start": "theia start",
    "start:watch": "concurrently --kill-others -n tsc,bundle,run -c red,yellow,green \"tsc -b -w --preserveWatchOutput\" \"theia build --watch --mode development\" \"yarn -s start\""
  },
```

And logs:
![image](https://github.com/eclipse-theia/theia/assets/59060212/43982f96-c5ca-4ff3-8f2c-169de3e65d9c)

**Comment by msujew** [comment]
Hey @znoliver,

do you have a reproducible example? It's hard to tell what is wrong, as there could be a lot of reasons why it doesn't work. Does `theia build --watch --mode development` work in general in your project? It doesn't seem to produce any output.

**Reply by znoliver** [reply]
yes, the example here: https://github.com/znoliver/theia-example
